Makefile

```diff
@@ -119,10 +119,15 @@ HTML_DST = \
   $(patsubst _extras/%.md,${DST}/%/index.html,$(sort $(wildcard _extras/*.md))) \
   ${DST}/license/index.html
 
+## * install-rmd-deps : Install R packages dependencies to build the RMarkdown lesson
+install-rmd-deps:
```

Comment:
I guess this is a new PHONY target, right?

Reply:
yes, thanks!